app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php

```diff
@@ -2,11 +2,15 @@
 
```

Comment:
Too much logic in GetPlaceCollectionPresenter

maybe this would help [gist](https://gist.github.com/nemoipaha/75a16feec79124d25bedc88bd4f2f28b)

Reply:
Created some presenters, but the are not static. Check it out, I'd like to hear your advice :thinking: